x-pack/plugins/monitoring/server/telemetry_collection/get_es_stats.ts

```diff
@@ -107,5 +128,17 @@ export interface ESClusterStats {
 export function handleElasticsearchStats(response: estypes.SearchResponse<ESClusterStats>) {
   const clusters = response.hits?.hits || [];
 
-  return clusters.map((cluster) => cluster._source!);
+  return clusters.map((cluster) => {
+    const clusterStats = cluster._source!;
+
+    // tolerate metricbeat monitoring structure
+    if (clusterStats.elasticsearch) {
+      clusterStats.cluster_stats = clusterStats.elasticsearch.cluster.stats;
+      clusterStats.cluster_uuid = clusterStats.elasticsearch.cluster.id;
+      clusterStats.cluster_name = clusterStats.elasticsearch.cluster.name;
+      clusterStats.elasticsearch = undefined;
```

Comment:
FWIW, I think it's OK to also ship this if it adds value. But we probably need to update the schema in `registerMonitoringTelemetryCollection` to declare it.

Reply:
Hm., it might be I am missing something here but I am not seeing a necessity to reflect this change in `registerMonitoringTelemetryCollection`. Current backfill (from the metricbeat data shape to application required shape) is an internal fetch player and guarantees to have data when we need [`cluster stats`](registerMonitoringTelemetryCollection).
If you please explain more in details which/where and why we need to reflect this change, to fill my ambiguous gap, I am happy to cover.